zqd/handlers_test.go

```diff
@@ -313,8 +322,69 @@ func postSpaceData(t *testing.T, c *api.Connection, spaceName, data string) *api
 		}
 		payloads = append(payloads, p)
 	}
-	last := payloads[len(payloads)-1].(*api.TaskEnd)
-	assert.Equal(t, last.Type, "TaskEnd")
-	assert.Nil(t, last.Error)
-	return s
+	return payloads
+}
+
+func TestPostLogs(t *testing.T) {
```

Comment:
I've been keeping all the `Test` functions above the utility functions to maintain readability. Are you cool with sticking to this convention?

Reply:
This doesn't affect readability either way for me, but I've changed it.